omnibridge/model_entities/text_to_image_model_entity.py

```diff
@@ -0,0 +1,27 @@
+from typing import Type
+
+from omnibridge.wrappers.wrapper_interfaces import ModelWrapper
+
+from omnibridge.model_entities.base_model_entity import BaseModelUnit
+from omnibridge.model_entities.models_io.base_model_io import ModelIO
+from omnibridge.model_entities.models_io.base_model_io import TextualIO, ImageIO
+
+
+class TextToImageModel(BaseModelUnit):
```

Comment:
Are we using this class somewhere?

Reply:
Not yet, would be used for DALLE and similar models

Reply:
@Liavatrix I think we wanted to remove the `ModelUnit` abstractions, right?